examples/Atlas/+atlasControllers/QPInput2D.m

```diff
@@ -0,0 +1,105 @@
+classdef QPInput2D
+% A container for the input to the InstantaneousQPController describing a
+% snapshot of the current plan at the current time and state. This is a "2D"
+% input because it reasons about the linear inverted pendulum in 2D (with a
+% fixed height assumption). We may eventually extend this to a QPInput3D
+% class.
```

Comment:
This threw me for a minute.  I assume that by 2D you mean x and y (+ constant z).  I thought you meant x+z, and perhaps had another copy for y+z.
maybe a better name is QPInputConstantHeight?